### PR TITLE
Rename admin replicate subcommands

### PR DESCRIPTION
**Pull request by poornas**
edit -> update
 remove -> rm

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

File: cmd/admin-replicate-rm.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ var adminReplicateRemoveFlags = []cli.Flag{
 }
 
 var adminReplicateRemoveCmd = cli.Command{
-	Name:         "remove",
+	Name:         "rm",
```
Comment (harshavardhana) [inline cmd/admin-replicate-rm.go:44]: keep and alias for `remove` @poornas for backward compatibility.

File: cmd/admin-replicate-rm.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ var adminReplicateRemoveFlags = []cli.Flag{
...
 	Usage:        "remove one or more sites from site replication",
```
Comment (harshavardhana) [inline cmd/admin-replicate-rm.go:45]:
```suggestion
	Usage:        "remove site(s) from site replication configuration",
```

File: cmd/admin-replicate-update.go [hunk 3]
```diff
@@ -40,13 +40,13 @@ var adminReplicateEditFlags = []cli.Flag{
 	},
 }
 
-var adminReplicateEditCmd = cli.Command{
-	Name:         "edit",
-	Usage:        "edit endpoint of site participating in cluster replication",
-	Action:       mainAdminReplicateEdit,
+var adminReplicateUpdateCmd = cli.Command{
+	Name:         "update",
```
Comment (harshavardhana) [inline cmd/admin-replicate-update.go:44]:
```suggestion
	Name:         "update",
```

Keep `edit` for backward compatibility reasons? - and update the examples to point to the new command.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by harshavardhana** [comment]
PTAL @poornas

File: cmd/admin-replicate-rm.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/admin-replicate-rm.go:44]: We could actually hide them @poornas

Comment (poornas) [reply cmd/admin-replicate-rm.go:44]: did explore that - looks like only flags can be hidden, not aliases

Comment (harshavardhana) [reply cmd/admin-replicate-rm.go:44]: Use the new minio/cli v1.24.1 version and use the `HiddenAliases: true` flag

Comment (poornas) [reply cmd/admin-replicate-rm.go:44]: the cli change does not work fully - it hides from the help as expected. However, it doesn't recognize the alias anymore

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
PTAL at the conflicts @poornas

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
PTAL at the conflicts @poornas

**Review by harshavardhana — APPROVED** [review]

**Review by krisis — APPROVED** [review]

**Pull request merged**